src/main/java/org/openRealmOfStars/game/States/CreditsView.java

```diff
@@ -588,19 +100,20 @@ public CreditsView(final ActionListener listener, final String title,
   * Generate credits and license text.
   * @param title Program Title
   * @param version Program Version
+  * @param authors Program Authors
   * @return Credit and license text
   * @throws IOException If error happens on reading license files.
   */
   public String generateCreditsAndLicense(final String title,
-      final String version) throws IOException {
+      final String version, final String authors) throws IOException {
     StringBuilder sb = new StringBuilder();
     sb.append("\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n"
         + "\n\n\n\n\n\n\n\n\n\n\n# ");
     sb.append(title);
     sb.append(" ");
     sb.append(version);
     sb.append("\n\n# Credits\n\n");
-    sb.append(MAIN_CREDITS);
+    sb.append(authors);
```

Comment:
I think this merge request is good idea, but now AUTHORS.md contains first line '# Authors of Open Realm Of Stars' Which makes sense in AUTHORS.md but I think it is kind of duplicate in credits view. What do you think about this one?

Reply:
Also add your name into AUTHORS.md under # Additional programming